### DISCUSSION TITLE
Notify when Renovate fails to run

### DISCUSSION THREAD

**Discussion by allanlewis** [renovatebot/renovate#26620] [Suggest an Idea]
### Tell us more.

Renovate occasionally fails to run for me due to errors on GitHub's side. I don't think this is something Renovate needs to handle any better, but it would be helpful if I could be notified of this, either via email or, even better Slack or some sort of Webhook.

**Comment by TimKnight-DWP** [comment]
Agreed - we have similar, running in GitLab occasionally we get authentication errors, this is something we need to fix by rotating the tokens etc, but the Job in GitLab shows as "Green" or Successful despite there being an error which prevented normal renovate execution.

It would be great to have a way we can fail the job in that scenario so we can alert on it and action things in advance of a user spotting it in the logs of an automated run
